packages/core/src/client/components/Plate.tsx

```diff
@@ -110,7 +116,7 @@ function PlateInner<
         plugins: pluginsProp as any,
       }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    []
+    [editorProp]
```

Comment:
Should we add this fix in the patch note?

Reply:
Sure, are you talking about https://github.com/udecode/plate/discussions/3338?

Reply:
Oh sorry, I got it. I've included this line by accident.